var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -116,7 +116,7 @@ class Boost(Package):
     variant('graph', default=False,
             description="Build the Boost Graph library")
 
-    depends_on('icu', when='+icu_support')
+    depends_on('icu4c', when='+icu_support')
```

Comment:
yeah -- the others look like that.  The description is also worded differently.  I'm being picky, I know.